Enigmatry.Entry.AspNetCore.Authorization/IAuthorizationProvider.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Enigmatry.Entry.AspNetCore.Authorization
+{
+    public interface IAuthorizationProvider
+    {
+        public bool HasRole(string roleName);
```

Comment:
Contract should be "public bool HasAnyRole(string[] roles)" and "public bool HasAnyPermission(string[] permissions).
We have use cases when user can have multiple roles and permissions assigned and without accepting array or roles and permissions we would not be able to support "OR" operator (e.g. this api endpoint is granted to user with either permission1 or permission2).

Reply:
I agree. I will add those tests